.travis.yml

```
@@ -7,7 +7,7 @@ env:
     - SALT_PILLARROOT=$TRAVIS_BUILD_DIR/tests/pillar/test-linux-main
     - SALT_REPO_URL=https://repo.saltproject.io/py3/redhat/7/x86_64/3003.repo
```

Update this to salt 3005...

```suggestion
    - SALT_REPO_URL=https://repo.saltproject.io/py3/redhat/7/x86_64/3005.repo
```